chatterbot/corpus/data/english/gossip.corpus.json

```diff
@@ -0,0 +1,108 @@
+{
+
+    "gossip": [
+        [
+            "GOSSIP",
```

Comment:
A number of these files have this extra single word at the top, it looks like it might be a duplicate of the outer corpus label. Either way, it is not a valid part of a conversation.